card/mod/admin/set/self/admin_info.rb

```diff
@@ -15,17 +15,17 @@ def clean_html?
     end
     return "" if warnings.empty?
     alert :warning, true do
-      render_warning_list warnings
+      warning_list warnings
     end
   end
 
-  def render_warning_list warnings
+  def warning_list warnings
```

Comment:
I think we should avoid format methods beginning with `render` that don't render views.  Eg, if someone tried to make a view named "warning list", it could be very confusing.